python-phoenixdb/README.rst

```diff
@@ -92,7 +92,7 @@ various Phoenix-specific features. In order to run the test suite, you need a
 working Phoenix database and set the ``PHOENIXDB_TEST_DB_URL`` environment variable::
 
     export PHOENIXDB_TEST_DB_URL='http://localhost:8765/'
-    nosetests
+    tox
```

Comment:
It is probably related to my local config, but it doesn't work for me locally, Does it work for you?
I think it is stuck after running `phoenixdb/tests/test_db.py s.`

Reply:
Talked to Istvan about it. WIth his patch: https://github.com/stoty/phoenix-queryserver/commit/b3e90c0d60dc718b8fdafdcf145709c656158fd1
it was working for me